### PR TITLE
Small cleanup April 2022

### PR DESCRIPTION
**Pull request by G-Lorenz**
The purpose of this pr is to collect various small cleanups: feel free to leave a comment proposing a new cleanup or a different approach to an existing one!

Commit messages are self-explanatory (at least for the first two cleanups... :-) )

### CODE REVIEW

**Comment by locutus2** [comment]
Correct indention for function `from_to` in types.h line 453 (one space is missing):

**Comment by niklasf** [comment]
https://github.com/official-stockfish/Stockfish/blob/e1f12aa4e61a8bbb772918c405137acdd85e3eec/src/syzygy/tbprobe.cpp#L1293

There are actually 462 combinations. Later, correctly:

https://github.com/official-stockfish/Stockfish/blob/e1f12aa4e61a8bbb772918c405137acdd85e3eec/src/syzygy/tbprobe.cpp#L802-L803

File: src/evaluate.cpp [hunk 1]
```diff
@@ -1083,13 +1083,13 @@ namespace {
...
-  bool useClassical = false;
+  bool useClassical = (pos.this_thread()->depth > 9 || pos.count<ALL_PIECES>() > 7) &&
+                      abs(eg_value(pos.psq_score())) * 5 > (856 + pos.non_pawn_material() / 64) * (10 + pos.rule50_count());
 
   // Deciding between classical and NNUE eval (~10 Elo): for high PSQ imbalance we use classical,
   // but we switch to NNUE during long shuffling or with high material on the board.
   if (  !useNNUE
-      || ((pos.this_thread()->depth > 9 || pos.count<ALL_PIECES>() > 7) &&
-          abs(eg_value(pos.psq_score())) * 5 > (856 + pos.non_pawn_material() / 64) * (10 + pos.rule50_count())))
+      || useClassical)
```
Comment (dubslow) [inline src/evaluate.cpp:1092]: I like this cleanup, makes code easier to read, but the comments describing the code should move with the code.

**Comment by vondele** [comment]
Small CI leftover:
```diff
diff --git a/README.md b/README.md
index 6e6e762e6..bc2b1b185 100644
--- a/README.md
+++ b/README.md
@@ -1,7 +1,6 @@
 ## Overview
 
 [![Build Status](https://github.com/official-stockfish/Stockfish/actions/workflows/stockfish.yml/badge.svg)](https://github.com/official-stockfish/Stockfish/actions)
-[![Build Status](https://ci.appveyor.com/api/projects/status/github/official-stockfish/Stockfish?branch=master&svg=true)](https://ci.appveyor.com/project/mcostalba/stockfish/branch/master)
 
 [Stockfish](https://stockfishchess.org) is a free, powerful UCI chess engine
 derived from Glaurung 2.1. Stockfish is not a complete chess program and requires a
```

**Comment by dav1312** [comment]
I'm not sure if this could be added to this PR but there is some inconsistent indentation in the [nnue_feature_transformer.h](https://github.com/official-stockfish/Stockfish/blob/22b7909809c731aea691184dd7c1a2b02c5946af/src/nnue/nnue_feature_transformer.h) and specially in the [Makefile](https://github.com/official-stockfish/Stockfish/blob/22b7909809c731aea691184dd7c1a2b02c5946af/src/Makefile)
https://github.com/official-stockfish/Stockfish/blob/22b7909809c731aea691184dd7c1a2b02c5946af/src/nnue/nnue_feature_transformer.h#L155-L156
https://github.com/official-stockfish/Stockfish/blob/22b7909809c731aea691184dd7c1a2b02c5946af/src/nnue/nnue_feature_transformer.h#L293
https://github.com/official-stockfish/Stockfish/blob/22b7909809c731aea691184dd7c1a2b02c5946af/src/Makefile#L825-L829

**Comment by Disservin** [comment]
`futility_move_count`  has its own function and is only used in one place, thus this can simply be removed
similar to this https://github.com/official-stockfish/Stockfish/compare/master...Disservin:removeFunc
maybe even leaving the `int futility_move_count` away

**Comment by pb00067** [comment]
For me the logic in probcut's TT condition is hard to follow:
```
             && !(   ss->ttHit
             && tte->depth() >= depth - 3
             && ttValue != VALUE_NONE
             && ttValue < probCutBeta))
```
Personally I would prefer this equivalent which corresponds more to human thinking
```
             && (   !ss->ttHit
             || tte->depth() < depth - 3
             || ttValue == VALUE_NONE
             || ttValue >= probCutBeta))
```

**Comment by dubslow** [comment]
> For me the logic in probcut's TT condition is hard to follow:

In either case, a grouped-together set of conditions should get more indentation for the operators within that group

**Comment by dsmsgms** [comment]
Stack->depth is simply not unused. Might as well remove the field from the struct.

**Comment by niklasf** [comment]
`README.md` refers to `Readme.md` (inconsistent casing)

**Comment by vondele** [comment]
I'll be merging this shortly, thanks for maintaining this PR.

**Comment by mstembera** [comment]
The comment in evaluate.cpp starting with "// Deciding between..." is now duplicated on lines 1086 and 1091 after this PR.

**Comment by vondele** [comment]
that was a merge mistake... will fix. Probably a good reason to start a new cleanup PR ;-)
